src/tests/JIT/Regression/JitBlue/Runtime_105723/Runtime_105723.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <!-- Needed for CLRTestEnvironmentVariable -->
+    <RequiresProcessIsolation>true</RequiresProcessIsolation>
+    <Optimize>True</Optimize>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="$(MSBuildProjectName).cs" />
+
+    <CLRTestEnvironmentVariable Include="DOTNET_TieredCompilation" Value="0" />
+  </ItemGroup>
+</Project>
```

Comment:
Nit: I think this can be simplified to just the minimums (e.g. https://github.com/dotnet/runtime/blob/main/src/tests/JIT/Regression/JitBlue/Runtime_105821/Runtime_105821.csproj)

Reply:
I originally had roughly that, but then running the test gives me:

```
❯ $CORE_ROOT/corerun /home/alahay01/dotnet/runtime_sve/artifacts/tests/coreclr/linux.arm64.Checked/JIT/Regression/JitBlue/Runtime_105723/Runtime_105723/Runtime_105723.dll
Unhandled exception. System.MissingMethodException: Entry point not found in assembly 'Runtime_105723, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null'.
[1]    3253248 abort (core dumped)  $CORE_ROOT/corerun
```

Reply:
It seemed like the `RequiresProcessIsolation` fixed that

Reply:
The regression tests normally run as part of a merged test assembly. If you want to build and run just an individual test you need to pass `/p:BuildAsStandalone=true` (or it can also be set as an environment variable, I think). So e.g. for regression tests I normally build them with
`.\dotnet.cmd build .\src\tests\JIT\Regression\JitBlue\Runtime_105721\Runtime_105721.csproj -c Release /p:BuildAsStandalone=true`.

Reply:
The `RequiresProcessIsolation` here has the side effect of always building the test as standalone, but it isn't necessary here and just requires some extra resources.

Reply:
>  you need to pass `/p:BuildAsStandalone=true`

That fixes it for me. Thanks. Updated PR.

Reply:
@jkoritzinsky @markples Where did we end up documenting this workflow?

Reply:
https://github.com/dotnet/runtime/blob/main/docs/workflow/testing/coreclr/testing.md#test-executors and https://github.com/dotnet/runtime/blob/main/docs/design/coreclr/jit/porting-ryujit.md are probably what you want